crates/api_models/src/refunds.rs

```diff
@@ -1,9 +1,12 @@
-use common_utils::{custom_serde, pii};
+use common_utils::pii;
 use serde::{Deserialize, Serialize};
 use time::PrimitiveDateTime;
 use utoipa::ToSchema;
 
-use crate::{admin, enums};
+use crate::{
+    admin,
+    enums::{self, Currency},
```

Comment:
Remove the import for `Currency`, qualify it with `enums::Currecy` wherever it's been used.